charts/ndb-operator/templates/deployment-ndb-operator-controller-manager.yaml

```diff
@@ -26,7 +26,7 @@ spec:
           - --health-probe-bind-address=:8081
           - --metrics-bind-address=127.0.0.1:8080
           - --leader-elect
-        image: {{ .Values.image.repository }}:{{ .Values.image.tag | default $.Chart.AppVersion }}
+        image: {{ .Values.image.repository }}:{{ .Chart.AppVersion }}
```

Comment:
same comment, check if you want to hardcode the repo URL as well

Reply:
Yes @mazin-s, as discussed, this should not be a user input at all. Let's say we release version `x` helm chart for the operator which uses the controller image version `y`. If the user has the ability to specify the image/version, they can change this to `y-1,2,3..` which can have undesired results. Best to hardcode this to the specific version for the chart.

Reply:
best practice is to let this as a value, and enforce correct version as default
there is use case with private registry where advanced customer want to have different tag

Reply:
Okay, to be clear- 
Something like : 
 image: {{ .Values.image.repository}}:v0.5.1?
 
 Good catch on the private registry use-case @tuxtof

Reply:
Are you saying this is good:
`{{ .Values.image.repository }}:{{ .Chart.AppVersion }}`
Or this:
`{{ .Values.image.repository }}:{{ .Values.image.tag | default $.Chart.AppVersion }}`

From my understanding you are saying that it's best to leave `{{ .Values.image.repository }}` as users might have their own private registry name they are using (this makes sense). With that same logic, would that also mean that we should leave the `.Values.image.tag` as they might also specify their own tag versions?

Reply:
yes both, they are case where they want to change the tag with -signed or -stream, because they modify the image

Reply:
Ok, addressed.